src/Runner.Worker/ExecutionContext.cs

```diff
@@ -539,6 +542,10 @@ public void AddIssue(Issue issue, string logMessage = null)
             }
 
             issue.Message = HostContext.SecretMasker.MaskSecrets(issue.Message);
+            if (issue.Message.Length > 4096)
```

Comment:
the change is unrelated to this PR...

We do trim issue messages from the service side to 4096 chars, so there is no reason we have to send out messages with more than that.